### PR TITLE
Multiple deployment in one account

### PR DESCRIPTION
**Pull request by patrick-239**
*Issue #52  available:*

*Description of changes:*
All recourse use UID and now it is possible to deploy several demos in one account

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by patrick-239** [comment]
Will try to find a workaround for mark name

File: README.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ To get the Retail Demo Store running in your own AWS account, follow these instr
 
 ### AWS Deployment Options
 
+***IMPORTANT NOTE:** Please use only letters in Cloud Formation Stack name.*
```
Comment (alexchirayath) [inline README.md:35]: Just curious, is there a specific reason for using just letters?

File: aws/cloudformation-templates/_template.yaml [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+    Type: String
+    Description: Amplitude API key for integrated product analytics and A/B testing results (optional).
+    NoEcho: true
+
+  OptimizelySdkKey:
+    Type: String
+    Description: Optimizely SDK key for experimentation (optional).
+    NoEcho: true
+
+  ParentStackName:
```
Comment (alexchirayath) [inline aws/cloudformation-templates/_template.yaml:93]: Is there a need to pass this as an additional parameter?
Could we maybe use psuedo parameters like AWS::?
More info [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html)

File: README.md (hunk 1 above)
Comment (patrick-239) [reply README.md:35]: unfortunately most of our services have a different naming rules. Problem with numbers comes from Lex (the support only letters) and as we define an IAM roles in CFN with resource based scope we have to define Lex name during deploy stage.

File: aws/cloudformation-templates/_template.yaml (hunk 2 above)
Comment (patrick-239) [reply aws/cloudformation-templates/_template.yaml:93]: Region could be received in any nested stack via pseudo parameter, but Parent stack name will be change every time when we use nested stack (we have many nested in nested ).

File: README.md (hunk 1 above)
Comment (alexchirayath) [reply README.md:35]: Got it, thanks!

File: aws/cloudformation-templates/_template.yaml (hunk 2 above)
Comment (alexchirayath) [reply aws/cloudformation-templates/_template.yaml:93]: Makes sense.

**Review by alexchirayath — APPROVED** [review]

**Comment by patrick-239** [comment]
Will cancel this pull request because:
1. Conflicts in different files. 
2. Macro name couldn't be unique

**Pull request closed without merging**